roundabout/templates/inventory/ajax_inventory_detail.html

```diff
@@ -143,6 +143,25 @@
                 data-location-id="{{ inventory_item.location.id}}"
                 data-node-type="{{ node_type }}">Edit Inventory Details</a>
 
+            {% if not user|has_group:"inventory only" %}
+                <a  id="add_coefficient_action"
+                    class="dropdown-item" 
+                    href="#"
+                    data-detail-url="{% url 'calibrations:event_valueset_add' inventory_item.id %}"
+                    data-node-id="{{ inventory_item.id }}"
+                    data-location-id="{{ inventory_item.location.id}}"
+                    data-node-type="calibration_event">Add Calibration Coefficients</a>
+
+                <a  id="add_conf_const_action"
+                    class="dropdown-item" 
+                    href="#"
+                    data-detail-url="{% url 'configs_constants:config_event_value_add' inventory_item.id %}"
+                    data-node-id="{{ inventory_item.id }}"
+                    data-location-id="{{ inventory_item.location.id}}"
+                    data-node-type="config_event}">Add Configuration/Constant Event</a>
```

Comment:
Nit: Suggest `Add Configuration/Constant Event` --> `Add Configurations/Constants` to use consistent cording with `Add Calibration Coefficients` above.

Reply:
Pushed changes look good.